CHANGELOG.md

```diff
@@ -9,6 +9,39 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.5.0 - 2019-03-25
+
+### Added
+
+- Add "networkPolicyOnly" as a new "encapsulation mode".
+- Support for running Antrea in [EKS] and [GKE] clusters; refer to the documentation.
+- New antctl "get" commands:
+   * in "agent mode": addressgroup, agentinfo, appliedtogroup, networkpolicy, podinterface
+   * in "controller mode": addressgroup, appliedtogroup, controllerinfo, networkpolicy
+- Support for a user-friendly "table" output format for antctl "get" commands.
+- Add health checks to Antrea components by leveraging the apiserver /healthz endpoint (both the Antrea Agent and Controller are running an apiserver).
+- Add documentation for troubleshooting connection issues to the Antrea Agent or Controller apiserver.
```

Comment:
I think it is not just about troubleshooting connection issues, but also connecting to apiserver to get all exposed state, and it is esp. helpful for checking the NetworkPolicy resources published from controller to agents.

Reply:
You're right. I forgot that this document talked about antctl as well.